src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+    private Person person;
+    private Tag tag;
+    private String type;
```

Comment:
I believe this is the operation. It can be better represented by an enum or even a boolean as there are only two operations, add/delete.